config/webpack/webpack.client.babel.ts

```diff
@@ -36,6 +36,8 @@ import { getenv } from '../../lib/getenv'
 
 import babelConfig from '../../babel.config'
 
+const WorkerPlugin = require('worker-plugin')
```

Comment:
Required for bundling webworker javascript modules.

Reply:
@abrie Can we change to `import`, for consistency?

Reply:
@ivan-aksamentov Yes!